packages/cli/src/commands/storybook.js

```diff
@@ -39,7 +39,7 @@ export const handler = ({ open, port, build, buildDirectory }) => {
   const staticAssetsFolder = path.join(getPaths().web.base, 'public')
   // Create the `MockServiceWorker.js` file
   // https://mswjs.io/docs/cli/init
-  execa(`yarn msw init "${staticAssetsFolder}"`, undefined, {
+  execa(`yarn msw init "${staticAssetsFolder}"`, ['--no-save'], {
```

Comment:
```suggestion
  execa(`yarn msw init --no-save "${staticAssetsFolder}"`, undefined, {
```

Reply:
this is what works locally for me. we'd need to update the syntax for exec otherwise.